webapp/utils/utils.jsx

```diff
@@ -1195,10 +1195,8 @@ export function canCreateCustomEmoji(user) {
         return false;
     } else if (window.mm_config.RestrictCustomEmojiCreation === 'admin') {
         // check whether the user is an admin on any of their teams
```

Comment:
Ah right, that's why the code was different there. This permissions logic never made sense.